core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/NamespaceSynchronizationConfig.java

```diff
@@ -331,6 +334,24 @@ public ConflictHandler getConflictHandler() {
     }
   }
 
+  public SyncFrequency getSyncFrequency() {
```

Comment:
[q] why is this public?

Reply:
getConflictHandler right above is public so I figured this should also be? Should I change this?

Reply:
If there is no good reason for it, it shouldn't be public.